src/api/app/models/user.rb

```diff
@@ -810,6 +811,10 @@ def tasks
     end
   end
 
+  def unread_notifications
+    notifications.not_marked_as_done.size
```

Comment:
Couldn't it be better to do a `.count` instead of a `.size`?

Reply:
`.size` calls `.count` or `.length` depending on the context. See this from [Stack Overflow](https://stackoverflow.com/questions/6083219/activerecord-size-vs-count).

Reply:
It seems that not, it is not better. Is the same: `.size` also throws a `SELECT COUNT(*) ...`.

Reply:
I want a `SELECT COUNT(*)` here and this is what `.size` does in this context. `.count` also does it.